cms/static/coffee/spec/main.coffee

```diff
@@ -27,9 +27,9 @@ requirejs.config({
         "moment": "xmodule_js/common_static/js/vendor/moment.min",
         "moment-with-locales": "xmodule_js/common_static/js/vendor/moment-with-locales.min",
         "text": "xmodule_js/common_static/js/vendor/requirejs/text",
-        "underscore": "xmodule_js/common_static/common/js/vendor/underscore",
-        "underscore.string": "xmodule_js/common_static/common/js/vendor/underscore.string",
-        "backbone": "xmodule_js/common_static/common/js/vendor/backbone",
+        "underscore": "common/js/vendor/underscore",
+        "underscore.string": "common/js/vendor/underscore.string",
+        "backbone": "common/js/vendor/backbone",
         "backbone.associations": "xmodule_js/common_static/js/vendor/backbone-associations-min",
```

Comment:
I've created a JIRA story to track this idea:

https://openedx.atlassian.net/browse/FEDX-181